Library/Homebrew/tap_constants.rb

```diff
@@ -2,6 +2,8 @@
 HOMEBREW_TAP_ARGS_REGEX = %r{^([\w-]+)/(homebrew-)?([\w-]+)$}
 # match taps' formulae, e.g. someuser/sometap/someformula
 HOMEBREW_TAP_FORMULA_REGEX = %r{^([\w-]+)/([\w-]+)/([\w+-.]+)$}
+# match core's formulae, e.g. homebrew/homebrew/someformula
+HOMEBREW_CORE_FORMULA_REGEX = %r{homebrew/homebrew/([\w+-.]+)$}
```

Comment:
`HOMEBREW_CORE_FORMULA_REGEX = %r{homebrew/homebrew/([\w+-.]+)$}i` to ignore the case.